Makefile

```diff
@@ -119,16 +120,16 @@ compose_build:
 	$(COMPOSE) build
 
 build_dev: ssl
-	docker buildx build --target dev -f experimenter/Dockerfile -t experimenter:dev experimenter/
+	docker buildx build $(BUILD_PLATFORMS) --target dev -f experimenter/Dockerfile -t experimenter:dev experimenter/
```

Comment:
Do you want to pull this whole thing into a `$(DOCKER_BUILD)` variable?

Reply:
You mean the `docker buildx build` part?